scripts/ibc/setup_ibc.sh

```diff
@@ -102,3 +105,17 @@ rly tx channel "$RELAYER_PATH"
 echo '# -------------------------------- IBC channel established ------------------------------- #'
 echo "Channel Information:"
 echo "$(rly q channels "$ROLLAPP_CHAIN_ID" | jq '{ "rollapp-channel": .channel_id, "hub-channel": .counterparty.channel_id }')"
+
+echo -e '--------------------------------- Set channel-filter --------------------------------'
+
+read -p "Enter the first channel: " channel1
```

Comment:
First and second channel are confusing. as one is hub and one is rollapp. 
it's best to take it automatically (you can see the previous command extracts the channels)